python/osrd_schemas/README.md

```diff
@@ -2,6 +2,15 @@
 
 This python library holds schemas definition used by both API and the RailJson Generator.
 
+## Node Types
+
+From now on, switch_types no longer exist in the db and are added as primitives in the code of services that need them:
+- In core we find them in `RJSSwitchType.java`.
+- In editoast they are found in `src/schema/switch_type.rs`.
+- In the fuzzer, we find them in `fuzzer.py`.
```

Comment:
The fuzzer only enumerates the switch types to regroup them in a dict. If we add a list of all switch primitives inside `osrd_schemas/switch_types.py` we wouldn't need to enumerate them in other places, which would make it unnecessary to update `fuzzer.py` on a primitive change, it could be removed from this list